.github/workflows/build-images-ci.yaml

```diff
@@ -198,6 +206,24 @@ jobs:
           cosign sign quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci@${{ steps.docker_build_ci_master_detect_race_condition.outputs.digest }}
           cosign sign quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci@${{ steps.docker_build_ci_master_unstripped.outputs.digest }}
 
+      - name: Generate SBOM
```

Comment:
Reading the commit message, I read that this step will add 10 minutes to the build. In particular here, this would be for every single CI image build. Am I misunderstanding because the commit message was written with another prior discussion in mind? Or is this true?